### DISCUSSION TITLE
Overdsipersion and zero inflation

### DISCUSSION THREAD

**Discussion by AlainZuur** [JenniNiku/gllvm#140] [General]
Hello,

I applied a Poisson gllvm (with covariates + latent variables + row random effect) on a data set with 80 species and 250 sites. 

Using the simulate(.., conditional = FALSE) function I created 10000 new data sets for each species, and fed that into createDHARMa(). That can obviously be improved by the simulation steps we discussed in a different thread. 

Bottom line is that I can apply things like testDispersion() and testZeroInflation() 80 times. Any ideas/suggestions on how to convert that into one test statistic for zero inflation or overdispersion? Obviously, applying a NB GLMM as well, and comparing AICs is an option....but not my preferred option. And obviously, it would be an illusion to expect that all 80 species would be fitted well.

Second question:
Those latent variables...they are the mechanism to capture correlation between species (c.q. columns). And in gllvm they are supposed to be iid, correct? So..how worried should I be if one of the latent variables shows a strong spatial pattern? I guess then it is time to extend the model with a spatial correlated row-random effect. But what if the latent variable still picks up something else...e.g. a strong temporal effect? How bad is that? Most people would look at those biplots and wonder what the axes represent (other than a mechanism to model correlation).

Alain

**Comment by BertvanderVeen** [comment]
Re. question 1, I suggest to ask this to Florian Hartig via the DHARMa github. I haven't really looked into the tests that DHARMa does, and do not have the time for it at the moment.

Re. question 2, see @fhui28 his publication on this subject: https://onlinelibrary.wiley.com/doi/full/10.1111/biom.13416. Note, @JenniNiku is working on implementing spatial/temporal structures for the LVs in gllvm.
